docs/data/sdks/typescript-react-native/index.md

```diff
@@ -19,6 +19,16 @@ The React Native SDK lets you send events to Amplitude. This library is open-sou
 
 --8<-- "includes/sdk-rn/rn-notification.md"
 
+## Compatibility Matrix
+
+The following matrix lists the support for Amplitude React Native SDK version for different versions of React Native and React Native CLI.
+
+| @amplitude/analytics-react-native | @react-native-community/cli | react-native      |Gradle|Android Gradle Plugin|
+|-----------------------------------|-----------------------------|-------------------|---|---|
+| >= 1.0.0                          | >= 10.0                     | >= 0.71           | 7.5.1+ | 7.2.1+ |
```

Comment:
I am wondering if the versions should be consistent with:
https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-react-native/package.json#L73
https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-react-native/android/build.gradle#L9

Reply:
Thanks. Fixed.